go.mod

```diff
@@ -110,7 +108,7 @@ require (
 	golang.org/x/net v0.0.0-20220630215102-69896b714898 // indirect
 	golang.org/x/tools v0.1.11 // indirect
 	golang.org/x/xerrors v0.0.0-20220609144429-65e65417b02f // indirect
-	google.golang.org/grpc v1.47.0 // indirect
+	google.golang.org/grpc v1.36.0 // indirect
```

Comment:
What made us update this dependency since the v0.20.0 release? In v0.20.0 we were using v1.31.1.

Reply:
You updated this in this pr: https://github.com/libp2p/go-libp2p/pull/1634/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R113

I think I picked 1.36 by looking at the go mod graph.

Reply:
Actually we can get rid of grpc dep completely. Need this first: https://github.com/libp2p/go-libp2p-testing/pull/63